.github/workflows/release.yml

```diff
@@ -4,6 +4,8 @@ on:
     tags:
       - '[0-9]+.[0-9]+.[0-9]+'
       - '[0-9]+.[0-9]+.[0-9]+-*'
+  pull_request:
```

Comment:
Ignore, we're just doing this to test it on the PR and will remove before merging.